rancher/concepts/index.md

```diff
@@ -63,6 +63,9 @@ Rancher implements a managed load balancer using HAProxy that can be manually sc
 
 Rancher implements a distributed DNS service by using its own light-weight DNS server coupled with a highly available control plane. Each healthy container is automatically added to the DNS service when linked to another service or added to a Service Alias. When queried by the service name, the DNS service returns a randomized list of IP addresses of the healthy containers implementing that service.
 
+* By default, all services within the same stack are added to the DNS service without requiring explicit links. If you need a custom DNS name for your service, that is different from your service name, you will be required to use a link to get the custom DNS name. 
+* To make services resolvable that are in different stacks, you will need to link them explicitly. 
+
```

Comment:
A couple of more things to add:

1) you can resolve containers within the same stack by their names
2) to link service to a load balancer, you would still need to create a link from LB to the target service.
3) Links are still needed if you have to put a service behind an Alias service.